parser-typechecker/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -1185,23 +1185,26 @@ displayTestResults :: Bool -- whether to show the tip
                    -> [(Reference, Text)]
                    -> [(Reference, Text)]
                    -> Pretty
-displayTestResults showTip ppe oks fails = let
-  name r = P.text (HQ.toText $ PPE.termName ppe (Referent.Ref r))
+displayTestResults showTip ppe oksUnsorted failsUnsorted = let
+  oks   = Name.sortByText fst [ (name r, msg) | (r, msg) <- oksUnsorted ]
+  fails = Name.sortByText fst [ (name r, msg) | (r, msg) <- failsUnsorted ]
+  name r = HQ.toText $ PPE.termName ppe (Referent.Ref r)
   okMsg =
     if null oks then mempty
-    else P.column2 [ (P.green "◉ " <> name r, "  " <> P.green (P.text msg)) | (r, msg) <- oks ]
+    else P.column2 [ (P.green "◉ " <> P.text r, "  " <> P.green (P.text msg)) | (r, msg) <- oks ]
```

Comment:
Suggest renaming `r` to almost anything other than `r` here and on line 1200 now that it's not a `Reference`.